data/css/_colors.scss

```diff
@@ -7,10 +7,3 @@ $accent-dark-color: #cc532b !default;
 
 $background-light-color: #e9e9e3 !default;
 $background-dark-color: #ddddd5 !default;
-
-$white-60-color: transparentize(white, 0.4) !default;
```

Comment:
This should be fine, he siad he only needed them for the blog template earlier becuase that's how the default theme worked earlier